tests/test_autonomy/test_cli/test_helpers/test_chain_helpers.py

```diff
@@ -43,6 +43,7 @@
 )
 
 
+@pytest.mark.skip
```

Comment:
Need to figure out a way to patch the class and methods from the `data/packages/valory/*/contract.py` modules